combat-trainer.lic

```diff
@@ -2056,7 +2056,7 @@ class TrainerProcess
         if DRStats.warrior_mage?
           bput("get my #{game_state.weapon_name}", 'You pick up')
         else
-          bput("remove my #{game_state.weapon_name}", 'from the air')
+          bput("get my #{game_state.weapon_name}", 'from the air')
```

Comment:
💡 Not a big deal but since the command is the same, you can move the 'from the air' to the messages for a generic 'get' command without a guild check

Reply:
Oh yeah, that's obvious now, lol.